public/pages/AnomalyCharts/utils/anomalyChartUtils.tsx

```diff
@@ -252,11 +259,12 @@ export const getAnomaliesHeatmapData = (
     dateRange
   );
 
-  entityAnomaliesMap.forEach((entityAnomalies, entity) => {
+  entityAnomaliesMap.forEach((entityAnomalies, entityListAsString) => {
     const maxAnomalyGradesForEntity = [] as number[];
     const numAnomalyGradesForEntity = [] as number[];
 
-    entityValues.push(entity);
+    entityStrings.push(entityListAsString);
```

Comment:
How about add some comments for `entityStrings` and `entityLists` ?

Reply:
Sure, will add.